LambdaEngine/Include/Resources/AnimationGraph.h

```diff
@@ -228,12 +248,22 @@ namespace LambdaEngine
 		// Returns true if the graph contains a transition with the specified from and to
 		bool HasTransition(const String& fromState, const String& toState);
 
+		// Returns 0xffffffff if not found
```

Comment:
This is subjective, isn't `UINT32_MAX` a bit friendlier for the eyes? If you do decide to change it, there's a duplicate of the comment below.

Reply:
Will fix